### PR TITLE
Feat/add grouping

### PR DESCRIPTION
**Pull request by ZergsLaw**

### CODE REVIEW

File: internal/api/config/lint.go [hunk 1]
```diff
@@ -104,114 +42,92 @@ func (cfg Config) buildFromUse() ([]lint.Rule, error) {
...
+	res = append(res, "PACKAGE_DIRECTORY_MATCH")
+	res = append(res, "PACKAGE_SAME_DIRECTORY")
+	return res
+}
 
-	if cfg.Lint.ServiceSuffix.Activated {
-		useRule = append(useRule, &cfg.Lint.ServiceSuffix.Value)
-	}
+func (cfg *Config) addBasic(res []string) []string {
+	res = append(res, "ENUM_FIRST_VALUE_ZERO")
```
Comment (hound672) [inline internal/api/config/lint.go:83]:
```
basicRules []string{"RULE1", "RULE2" ...}
return res.append(res, ...basicRules)
```
?

File: internal/lint/lint.go [hunk 2]
```diff
@@ -25,10 +25,10 @@ type Rule interface {
...
 // New creates a new Lint.
-func New(rules []Rule, rootPath string, excludesDirs []string) *Lint {
+func New(rules []Rule, rootPath string, ignoreDirs []string) *Lint {
 	return &Lint{
-		rules:        rules,
-		rootPath:     rootPath,
-		excludesDirs: excludesDirs,
+		rules:      rules,
+		rootPath:   rootPath,
+		ignoreDirs: ignoreDirs,
```
Comment (hound672) [inline internal/lint/lint.go:32]: Why `ingore` instead of `exclude`?
`ingore` key work used in buf?

**Comment by hound672** [comment]
Can you add new rules to example config?

File: internal/lint/lint.go (hunk 2 above)
Comment (ZergsLaw) [reply internal/lint/lint.go:32]: Yeah, only because of buf

**Pull request merged**